diff --git a/node_modules/svelte-awesome-color-picker/ColorPicker.svelte b/node_modules/svelte-awesome-color-picker/ColorPicker.svelte
index 4695246..395dd9b 100644
--- a/node_modules/svelte-awesome-color-picker/ColorPicker.svelte
+++ b/node_modules/svelte-awesome-color-picker/ColorPicker.svelte
@@ -48,14 +48,14 @@ function getComponents() {
 let button;
 let wrapper;
 function mousedown({ target }) {
-    if (isInput) {
-        if (button.isSameNode(target)) {
-            isOpen = !isOpen;
-        }
-        else if (isOpen && !wrapper.contains(target)) {
-            isOpen = false;
-        }
-    }
+    // if (isInput) {
+    //     if (button.isSameNode(target)) {
+    //         isOpen = !isOpen;
+    //     }
+    //     else if (isOpen && !wrapper.contains(target)) {
+    //         isOpen = false;
+    //     }
+    // }
 }
 $: {
     if (color && isHsv(color)) {
diff --git a/node_modules/svelte-awesome-color-picker/util/convert.js b/node_modules/svelte-awesome-color-picker/util/convert.js
index e0a4f73..d4fdb9e 100644
--- a/node_modules/svelte-awesome-color-picker/util/convert.js
+++ b/node_modules/svelte-awesome-color-picker/util/convert.js
@@ -32,10 +32,10 @@ function rgb2hex({ r, g, b, a = 1 }) {
 function hex2rgb(hex) {
     const h = hex.hex;
     return {
-        r: parseInt(h.substring(1, 2), 16),
-        g: parseInt(h.substring(3, 2), 16),
-        b: parseInt(h.substring(5, 2), 16),
-        a: h.length <= 7 ? 1 : parseInt(h.substring(7, 2), 16) / 255
+        r: parseInt(h.substr(1, 2), 16),
+        g: parseInt(h.substr(3, 2), 16),
+        b: parseInt(h.substr(5, 2), 16),
+        a: h.length <= 7 ? 1 : parseInt(h.substr(7, 2), 16) / 255
     };
 }
 /**
